audit/projects/kubernetes-public/services/bigquery/bigquery.datasets.k8s_triage.access.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "role": "WRITER",
+    "userByEmail": "k8s-triage@k8s-infra-prow-build-trusted.iam.gserviceaccount.com"
+  },
+  {
+    "role": "OWNER",
+    "groupByEmail": "k8s-infra-prow-oncall@kubernetes.io"
+  }
+]
```

Comment:
It's interesting to me that https://github.com/kubernetes/k8s.io/pull/2461 explicitly _does not_ set access, opting to just use IAM bindings.  Yet the BQ API still acts as if access has been set on the dataset directly?